shell/src/main/java/alluxio/cli/fs/command/StatCommand.java

```diff
@@ -55,6 +56,14 @@ public final class StatCommand extends AbstractFileSystemCommand {
           .required(false)
           .desc("specify a file by file-id")
           .build();
+  private static final Option OMIT_MOUNT_INFO =
+      Option.builder("m")
```

Comment:
the short option is confusing...
m as in mount? can probably use "xm" as in eXclude Mount info instead

Reply:
https://github.com/Alluxio/alluxio/pull/16893

It is consistent with this. He said to use this. @jiacheliu3 .hh